rbc_base/MotorState[3] states